### PR TITLE
OpenAPI validation

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by fao89** [comment]
@daviddavis could you please review this PR and merge https://github.com/pulp/pulpcore/pull/1065 ?

**Review by daviddavis — DISMISSED** [review]

**Comment by daviddavis** [comment]
Was there a bug fixed here and if so would it be possible to get a changelog entry?

**Comment by fao89** [comment]
> Was there a bug fixed here and if so would it be possible to get a changelog entry?

I can do it

File: CHANGES/8037.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix OpenAPI schema pagination
```
Comment (daviddavis) [inline CHANGES/8037.bugfix:1]: Any chance you could provide a little more context? I'm not sure I understand the problem that was fixed.

File: pulp_ansible/app/galaxy/v3/pagination.py [hunk 2]
```diff
@@ -73,27 +73,33 @@ def get_paginated_response_schema(self, schema):
             "type": "object",
             "properties": {
                 "meta": {
-                    "count": {
-                        "type": "integer",
-                        "example": 123,
+                    "type": "object",
```
Comment (fao89) [inline pulp_ansible/app/galaxy/v3/pagination.py:76]: @daviddavis customized pagination for v3 had an issue with the schema, it was not following OpenAPI standard

Comment (daviddavis) [reply pulp_ansible/app/galaxy/v3/pagination.py:76]: So like `Fixed v3 schema pagination to match OpenAPI standard` would be great.

**Review by daviddavis — APPROVED** [review]

**Pull request merged**